### PR TITLE
[FIX] mail: keyboard shouldn't pop on WebRTC call on mobile devices

### PR DESCRIPTION
**Pull request by res-odoo**
Steps to reproduce:

1. Log on odoo with a user A on mobile device
2. Log on odoo with a user B on desktop
3. User B call user A
4. User A answer the call and the keyboard popup on the screen => BUG

To avoid this behavior, we disable the input focus on mobile device when
 the user answer a call.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/static/src/models/discuss/discuss.js [hunk 1]
```diff
@@ -221,7 +221,9 @@ function factory(dependencies) {
             this.update({
                 thread: link(thread),
             });
-            this.focus();
+            if (!this.env.device.isMobileDevice) {
+                this.focus();
+            }
```
Comment (seb-odoo) [inline addons/mail/static/src/models/discuss/discuss.js:226]: Discuss component is not used to display threads in mobile, this should be in chat window instead no?

I guess it doesn't hurt to have it here too, but it likely doesn't resolve the issue.

Comment (res-odoo) [reply addons/mail/static/src/models/discuss/discuss.js:226]: I added the check inside chat window, feel you free to merge when it's ok for you :+1:

**Comment by seb-odoo** [comment]
Thank you for the update and sorry for being late on the merge, now this will need to be rebased on v15 and the PR retargetted (PR back to draft during the operation to avoid code owners spam).

I'll do it tomorrow at a more reasonable time, or if you see this message before I'm able to do it and have some time, feel free to do it too.

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**